### DISCUSSION TITLE
Coolapk UWP v3.0.40

### DISCUSSION THREAD

**Discussion by wherewhere** [Coolapk-UWP/Coolapk-UWP#168] [Announcements]
###### 更新日志
1. 更新版本号至 v3.0.40
2. 重新编写了程序
###### 下载链接
[![Microsoft Store](https://img.shields.io/badge/download-%e4%b8%8b%e8%bd%bd-magenta.svg?label=Microsoft%20Store&logo=Microsoft&style=for-the-badge&color=11a2f8)](https://www.microsoft.com/store/apps/9N0DMXZVMQVL "Microsoft Store")
###### 屏幕截图
![主页](https://raw.githubusercontent.com/Tangent-90/Coolapk-UWP/screenshots/Screenshots/v3.0.40/Snipaste_2023-04-17_18-30-26.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/Coolapk-UWP/Coolapk-UWP/releases/tag/v3.0.40'>Coolapk UWP v3.0.40</a>.</em>

**Comment by gz16szh** [comment]
服务器错误 514：Frequently Capped

**Reply by 6Leokk** [reply]
请问作者不维护这个项目了吗？

**Comment by gz16szh** [comment]
是的吧



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2023年09月29日 15:45 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [Coolapk-UWP/Coolapk-UWP] Coolapk UWP v3.0.40 (Discussion #168) |

请问作者不维护这个项目了吗？
